Assets/BossRoom/Scripts/Shared/ApplicationController.cs

```diff
@@ -109,7 +109,11 @@ public void QuitGame()
             }
             else
```

Comment:
Not exactly related to this specific issue, but it might help prevent some other bugs if we replaced this if-else with two methods instead, since the only place where this is now called (UIQuitPanel) knows about that condition and knows what outcome it wants based on it.

Reply:
good catch, fixed